### PR TITLE
[FIX] account_payment_purchase: Propagate payment_mode on MTO/drop shipping

### PR DESCRIPTION
**Pull request by pedrobaeza**

### CODE REVIEW

**Comment by rvalyi** [comment]
:+1:  LGTM

File: account_payment_purchase/models/procurement_order.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, api
```
Comment (elicoidal) [inline account_payment_purchase/models/procurement_order.py]: sort order

File: account_payment_purchase/models/__init__.py [hunk 2]
```diff
@@ -22,3 +22,4 @@
 
 from . import purchase_order
 from . import stock_picking
```
Comment (elicoidal) [inline account_payment_purchase/models/__init__.py:25]: import order

**Comment by pedrobaeza** [comment]
@elicoidal, fixed!

I don't want coveralls to be sad, so I have developed the full test suite for this module :wink:

**Comment by elicoidal** [comment]
LGTM :)

**Comment by rafaelbn** [comment]
Thanks! :+1:

**Comment by pedrobaeza** [comment]
Can any of you merge this with the 3 approvals?

**Pull request merged**